packages/contracts-bedrock/test/L2/L2StandardBridgeInterop.t.sol

```diff
@@ -0,0 +1,361 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Target contract is imported by the `Bridge_Initializer`
+import { Bridge_Initializer } from "test/setup/Bridge_Initializer.sol";
+import { console2 } from "forge-std/console2.sol";
+
+// Target contract dependencies
+import {
+    L2StandardBridgeInterop,
+    InvalidDecimals,
+    InvalidLegacyAddress,
+    InvalidSuperchainAddress,
+    InvalidTokenPair,
+    IOptimismMintableERC20Factory,
+    MintableAndBurnable
+} from "src/L2/L2StandardBridgeInterop.sol";
+import { IERC20Metadata } from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import { IOptimismMintableERC20 } from "src/universal/IOptimismMintableERC20.sol";
+
+// TODO: Replace Predeploys.OPTIMISM_SUPERCHAIN_ERC20_FACTORY with optimismSuperchainERC20Factory
```

Comment:
This TODO is for the future, right?

Reply:
Yes, until we have the superchain factory implementation